src/new-client/src/services/currencyPairs/types.ts

```diff
@@ -10,7 +10,7 @@ export interface RawCurrencyPairUpdate extends CollectionUpdate {
   CurrencyPair: CurrencyRaw
 }
 
-export interface RawCurrencyPairUpdates extends CollectionUpdates {
```

Comment:
We do this for Trades too.  Was there a prior world in which the API also sent diffs/updates instead of state-of-the-world? @garyadaptive?

Reply:
I think that this has to do with the way that the old API was working. The thing is that -at least for the currencyPairs- we are not taking those fields into account, and the code of the current client doesn't take them into account either, so I thought that I would just remove it... That makes the mocks easier to build and I don't see the point in keeping stuff in the interface that we ignore.